docs/operating-scylla/admin-tools/scylla-sstable.rst

```diff
@@ -500,6 +500,7 @@ The content is dumped in JSON, using the following schema:
         "scylla_build_id": String
         "scylla_version": String
         "ext_timestamp_stats": {"$key": int64, ...}
+        "sstable_identifier": String, // UUID
```

Comment:
Why is it a string and not a uuid?

Reply:
It is stored as a uuid.
I just followed `run_identifier`'s example in the documentation.

Reply:
Then it's misleading. String -> formatted UUID.

Reply:
This is documenting a JSON schema and JSON doesn't have a UUID type, so it has to be converted to string.